torch_xla/csrc/lowering_context.h

```diff
@@ -56,6 +57,9 @@ class LoweringContext {
   xla::StatusOr<xla::XlaComputation> Build(const xla::XlaOp& root);
 
  private:
+  void LowerPostOrder(
```

Comment:
Can this be noexcept?

Reply:
We never use to quality APIs by exception throwing semantics, I am not sure why should we start here.